### PR TITLE
`elementwise` fix up

### PR DESCRIPTION
**Pull request by rush2406**
- modified container instance methods
- fixed tests

### CODE REVIEW

**Comment by rush2406** [comment]
@VedPatwardhan are all of the elementwise functions required to be added into `array/__init__.py`?

File: ivy/array/elementwise.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# for review
```
Comment (djl11) [inline ivy/array/elementwise.py:1]: could you please add the `ivy.Array instance method variant of ivy.<>....` docstrings for all instance methods? Could you please also add all parameters into the docstrings, as explained [here](https://github.com/unifyai/ivy/blob/d8f8fe6986145051e50816fe7bea33e92d997a86/docs/partial_source/deep_dive/11_docstrings.rst#docstrings)?

File: ivy/container/elementwise.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# for review
```
Comment (djl11) [inline ivy/container/elementwise.py:1]: could you please add the `ivy.Array instance method variant of ivy.<>....` docstrings for all instance methods? Could you please also add all parameters into the docstrings, as explained [here](https://github.com/unifyai/ivy/blob/d8f8fe6986145051e50816fe7bea33e92d997a86/docs/partial_source/deep_dive/11_docstrings.rst#docstrings)?

**Review by djl11 — CHANGES_REQUESTED** [review]
see comments :)

**Pull request merged**